src/local.css

```diff
@@ -25,7 +25,7 @@ body {
 }
 h1 {
   text-indent:0px;
-  font-family: 'Montserrat', cursive;
+  font-family: 'Montserrat', bold;
```

Comment:
As previously commented, bold is AFAIK not a font family.
What is making the whole thing bold here?